OptimoveSDK/optimove-sdk/src/main/java/com/optimove/android/Optimove.java

```diff
@@ -79,10 +79,9 @@ public enum IBeaconProximity {
 
     private Optimove(@NonNull Context context, OptimoveConfig config) {
         this.context = context;
-        this.userInfo = UserInfo.newInstance(context);
```

Comment:
Will it cause getVisitorId API and setUserEmail API's to crash when Optimove is not configured?

Reply:
true, reverted to always setting it